privileges-grantor/README.md

```diff
@@ -49,38 +60,53 @@ The jars can be downloaded from the link provided above and uploaded to S3 via t
 variable "pg_lambda_version" {
   description = "Version of the Privilege Grantor Lambda."
   type        = "string"
-  default     = "4.2.0"
+  default     = "6.1.0"
```

Comment:
Need to mention this default version change in the changelog too.

Reply:
Ok.